src/Sylius/Component/Attribute/tests/Model/AttributeValueTest.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\Component\Attribute\Model;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Component\Attribute\Model\AttributeInterface;
+use Sylius\Component\Attribute\Model\AttributeSubjectInterface;
+use Sylius\Component\Attribute\Model\AttributeValue;
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+
+class AttributeValueTest extends TestCase
+{
+    private AttributeValue $attributeValue;
+
+    protected function setUp(): void
+    {
+        $this->attributeValue = new AttributeValue();
+    }
+
+    public function testCanBeInstantiated(): void
+    {
+        self::assertInstanceOf(AttributeValue::class, $this->attributeValue);
+    }
+
+    public function testShouldImplementAttributeValueInterface(): void
+    {
+        self::assertInstanceOf(AttributeValueInterface::class, $this->attributeValue);
+    }
+
+    public function testShouldHasNoIdByDefault(): void
+    {
+        self::assertNull($this->attributeValue->getId());
+    }
+
+    public function testShouldDoesNotBelongToASubjectByDefault(): void
```

Comment:
```suggestion
    public function testDoesNotBelongToASubjectByDefault(): void
```
the `Should` part could be removed from these methods names